ml-algorithms/src/test/java/org/opensearch/ml/engine/httpclient/MLHttpClientFactoryTests.java

```diff
@@ -36,7 +36,9 @@ public void test_getSdkAsyncHttpClient_success() {
     @Test
     public void test_validateIp_validIp_noException() throws Exception {
         MLHttpClientFactory.validate(HTTP, TEST_HOST, 80, PRIVATE_IP_DISABLED);
+        MLHttpClientFactory.validate(HTTPS, TEST_HOST, 80, PRIVATE_IP_DISABLED);
```

Comment:
HTTPS should be 443, why it's allowing 80 here?

Reply:
If we pass port as -1, then it will reset to 443, otherwise it accepts all the port values between 0-65536 (according to validation logic in the code).

Reply:
Also changed all HTTPS ports to 443 now